BlazarUI/app/stylus/components/branch-state/commits-summary.styl

```diff
@@ -19,42 +24,57 @@
 
   &:not(.commits-summary-popover__commit-list--single-commit):before
     position absolute
-    top 30px
+    top 49 px
     bottom 0
-    left 12px
+    left 11px
     width 1px
     content ""
-    background-color $color-gypsum
+    background-color $color-eerie
     z-index -1
 
 .commits-summary-poverover__commit-icon
   width 24px
   text-align center
   vertical-align top
-  padding-top 6px
+  padding-top 7px
+  color $color-eerie
 
 .commits-summary-poverover__commit-icon--first
   vertical-align middle
   padding-top 0
 
 .commits-summary-poverover__branch-head-icon
   padding-bottom 2px
-  background-color $color-heffalump
+  position relative
+  left 0.5px
+
+  &:before
```

Comment:
I was able to get rid of the branch-head svg and just rely on the octicon and css instead. This should eliminate the brief loading delay that I would sometimes see when triggering the previous tooltip for the first time.